packages/fxa-auth-server/lib/senders/templates/multiSubscriptionsPaymentExpired.txt

```diff
@@ -0,0 +1,17 @@
+{{{subject}}}
+
+{{{t "Your credit card is about to expire" }}}
+
+{{{t "The credit card you're using to make payments for the following subscriptions is about to expire." }}}
+
+{{#subscriptionProductNames}}
+  - {{{t . }}}
```

Comment:
Don't need `{{{t }}}` here, since `.` is not a localizable string.

Reply:
🤦